lib/services/network/downloads_service.dart

```diff
@@ -110,20 +111,13 @@ class AttachmentDownloadController extends GetxController {
 
       error.value = true;
       attachmentDownloader._removeFromQueue(this);
-      return Response(requestOptions: RequestOptions(path: ''));
+      return null;
```

Comment:
doing this on a catchError is bad, the catchError expects to return the same type that the regular function would

Reply:
I honestly had no clue how to fix that, dart gave me a warning.